apps/frontend/src/app/1_atoms/DappLocked/DappLocked.tsx

```diff
@@ -3,22 +3,29 @@ import React, { FC } from 'react';
 import { t } from 'i18next';
 
 import { SovrynLogo } from '../../2_molecules';
+import { NetworkPicker } from '../../2_molecules/NetworkPicker/NetworkPicker';
 import sovrynLogo from '../../../assets/images/sovryn-small-logo.svg';
 import { translations } from '../../../locales/i18n';
 
 export const DappLocked: FC = () => (
-  <div className="flex-grow bg-black flex items-center justify-center">
-    <div className="sm:rounded sm:w-96 sm:px-14 sm:py-10 sm:bg-gray-90">
-      <SovrynLogo
-        image={sovrynLogo}
-        className="w-8 mx-auto mb-5"
-        dataAttribute="header-logo"
-      />
-      <div className="max-w-xs mx-auto text-center text-sm leading-4 mb-1">
-        <div className="mb-6">
-          {t(translations.maintenanceMode.fullD2.title)}
+  <div className="flex-grow bg-black flex flex-col">
+    <div className="w-full flex justify-end p-4">
+      <NetworkPicker className="mr-6" />
```

Comment:
There's a problem with `NetworkPicker` - it's cut out.

<img width="1054" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/63608a6a-833d-4816-b6e5-f1d0cea476cc">